invenio_rdm_records/resources/serializers/bibtex/schema.py

```diff
@@ -0,0 +1,194 @@
+            bib_fields = getattr(BibTexFormatter, format)()
+        else:
+            bib_fields = BibTexFormatter.other()  # Default parser fields
+
+        out = "@" + bib_fields.pop("name") + "{"
+        out += self._get_citation_key(data) + ",\n"
+        inp = self._fetch_fields(data, bib_fields)
+        out += self._clean_input(inp)
+        out += "}"
+        return out
+
+    def _fetch_fields(self, data, bib_fields):
+        req_fields = bib_fields["req_fields"]
+        non_req_fields = bib_fields.get("opt_fields", []) + ["doi", "url"]
+
+        # The following fields are taken from Zenodo for consistency/compatibility reasons
+        fields = {
+            "address": data.get("locations", None),  # Done
+            "author": data.get("creators", None),  # Done
+            "publisher": (
+                lambda publishers: None if publishers is None else publishers[0]
+            )(
+                data.get("publishers", None)
+            ),  # Done
+            "title": (lambda titles: None if titles is None else titles[0])(
+                data.get("titles", None)
+            ),  # Done
+            "year": data.get("date_created", {}).get("year", None),  # Done
+            "doi": data.get("doi", None),  # Done
+            "month": data.get("date_created", {}).get("month", None),  # Done
+            "version": data.get("version", None),  # Done
+            "url": (lambda doi: None if doi is None else "https://doi.org/" + doi)(
+                data.get("doi", None)
+            ),  # Done
+            "school": data.get(
+                "school", None
+            ),  # [TODO] Implement once thesis is merged
+            "journal": data.get(
+                "journal", None
+            ),  # [TODO] Implement once journal is merged
+            "volume": data.get(
+                "volume", None
+            ),  # [TODO] Implement once journal is merged
+            "booktitle": data.get(
+                "booktitle", None
+            ),  # [TODO] implement book optional params
+            "number": data.get("number", None),  # [TODO] implement book optional params
+            "pages": data.get("pages", None),  # [TODO] implement book optional params
+            "note": data.get("note", None),  # [TODO] Implement once notes are merged
+            "venue": data.get(
+                "venue", None
+            ),  # [TODO] Zenodo backward compatibility issue
+        }
+
+        out = ""
+        for field in req_fields + non_req_fields:
+            value = fields[field]
+            if value is not None:
+                out += self._format_output_row(field, value)
+            elif value is None and value in req_fields:
+                raise ValueError("Required value not found: {0}".format(value))
+
+        return out
+
+    def _format_output_row(self, field, value):
+        # [TODO] Refractor old zenodo code
+
+        out = ""
+        if isinstance(value, six.string_types):
```

Comment:
six was used for Python 2 and 3 compatibility, now we can replace this by `str` (remove the import too)

```suggestion
        if isinstance(value, str):
```